src/App.js

```diff
@@ -17,9 +47,10 @@ class App extends Component {
         </div>
         
         <div className="seq">
-
-          <Grid className="grid" />
-          <Grid className="grid" />
+          <Grid
+            className="grid"
+            toggleButton={this.toggleButton}
```

Comment:
This is a common pattern: we're passing down the state object to child components, and also functions that operate on that state.